exercises/spiral-matrix/spiral_matrix_test.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+
+from spiral_matrix import spiral
+
```

Comment:
Could you add a comment that this tests are based on version `1.0.0` of https://github.com/exercism/problem-specifications/blob/master/exercises/spiral-matrix/canonical-data.json?